src/CoreWCF.Queue/src/CoreWCF/Queue/Common/QueueTransportContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CoreWCF.Channels;
+using CoreWCF.Configuration;
+using CoreWCF.Queue.Common.Configuration;
+
+namespace CoreWCF.Queue.Common
+{
+    public class QueueTransportContext
+    {
```

Comment:
All these properties should be internal set.

Reply:
Property ServiceDispatcher required [here](https://github.com/Ximik87/CoreWCF/blob/main/src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqQueueTransport.cs#L97), can we do it differently?

Reply:
thanks, I misunderstood